src/rep.rs

```diff
@@ -460,8 +508,12 @@ pub struct History {
     #[serde(deserialize_with = "datetime_from_unix_timestamp")]
     pub created: DateTime<Utc>,
     #[cfg(not(feature = "chrono"))]
-    pub created: u64,
+    pub created: i64,
     pub created_by: String,
+    // Not Required
+    pub tags: Option<Vec<String>>,
+    pub size: i64,
+    pub comment: String,
```

Comment:
I think the "Not Required" comment should be removed. If it's just about the `tags` field then I think that `Option` conveys that but if its for `size` or `comment` then it's confusing because they're not `Option<T>`

Reply:
In [Docker doc](https://docs.docker.com/engine/api/v1.41/#operation/ImageHistory) `Tags` field are required, but it's not true (I checked it locally). I wanted to point this out so that other developers don't think this is a mistake. Do you think it is not necessary?

Reply:
I don't think pointing it out makes sense as this is the case for a lot of fields on most of the structs here. The api is very loose on saying what actually is Nullable and what's not. 

When I added missing fields to ContainerDetails I basically made all fields required where it said and then I inspected a container without any filters and options and ensured it was a container that has no volumes/networks etc. After that it is trial and error with serde deserialization, and adjusting fields to options where it errors out.

Reply:
> The api is very loose on saying what actually is Nullable and what's not.

Agree with you.

>     // Not Required
Removed it.